jenkins/bookkeeper-master-job-configuration.yaml

```diff
@@ -0,0 +1,60 @@
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+- scm:
+    name: repo
+    scm:
+    - git:
+        url: https://git-wip-us.apache.org/repos/asf/bookkeeper.git
+        branches:
+        - "*/master"
+- job:
+    name: bookkeeper-master
+    description: "Nightly build for BookKeeper master.\r\nThis job is built using jenkins-jobs build scripts committed on BookKeeper repo.\r\n\r\nNo not edit manually this job"
+    project-type: maven
+    jdk: "JDK 1.8 (latest)"
+    node: Hadoop
+    maven:
+      root-pom: pom.xml
+      goals: clean apache-rat:check package findbugs:check -Dmaven.test.failure.ignore=false deploy
```

Comment:
This is the.current configuration.
Maybe we should create two jobs ?
 I think @nicoloboschi will add the.job for pr